docs/site/submitting_a_pr.md

```diff
@@ -0,0 +1,629 @@
+---
+title: Submitting a pull request to Loopback 4
+lang: en
+keywords: contributing, LoopBack community, pull request, PR, loopback
+sidebar: contrib_sidebar
+permalink: /doc/en/lb4/submitting_a_pr.html
+---
+
+## How to submit a pull request (PR) for Loopback 4
+
+Here is a tutorial on how to submit a pull request (PR) for loopback v4.
+
+### 1. Log in to github
```

Comment:
Before we dive into the details, we should outline the steps for two levels:

- github newbies
  - (detailed steps)
- github experts
  - (loopback-next specifics)

Reply:
@raymondfeng , I haven't yet created the "expert" list yet. I have mainly addressed all the other comments. But I will address it soon.